definitions/variable/water/water.yaml

```diff
@@ -31,6 +35,10 @@
     definition: Water consumption for irrigation
     unit: km3/yr
     tier: 2
+- Water Consumption|Electricity:
+    definition: Total water withdrawals for net electricity production
+    unit: km3/yr
+    tier: 2
```

Comment:
```suggestion
- Water Consumption|Electricity:
    definition: Total water withdrawals for total electricity generation
    unit: km3/yr
    tier: 2
- Water Consumption|Electricity|{Electricity Source}:
    definition: Total water withdrawals for electricity generation from {Electricity Source}
    unit: km3/yr
    tier: 2
```

Reply:
This would require adding a "tag_electricity-source.yaml", right?

Reply:
That's the beauty of the tag-lists...
This particular one already exists and this way, we can make sure that Secondary Energy|Electricity|..., Water Consumption|Electricity|..., technical parameters... all use the same elecitricity-generation technologies. https://github.com/IAMconsortium/common-definitions/blob/main/definitions/variable/energy/tag_secondary_electricity_sources.yaml

Reply:
For context, the location of any files within the "variable" folder does not have any functional impact, sub-folders are just to help us humans structure the info. For the code, it's all the same, so you can reuse across sub-folders.